benchmarks/Exporter/NonExporterBenchmarks.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using BenchmarkDotNet.Attributes;
+
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Trace.Configuration;
+using OpenTelemetry.Trace.Export;
+
+namespace Benchmarks.Exporter
+{
+    /// <summary>
+    /// These benchmarks compare OpenTelemetry without an Exporter vs with a NoOp Exporter
+    /// </summary>
+    /// <remarks>
+    /// To run: .\Benchmarks.exe --filter NonExporterBenchmarks
+    /// </remarks>
+    [MemoryDiagnoser]
+#if !NET462
+    [ThreadingDiagnoser]
+#endif
+    public class NonExporterBenchmarks
+    {
+
+        [Params(1, 10)]
```

Comment:
I suppose a larger number of spans or traces is better to reduce the impact of initialization like `TracerFactory.Create` and `tracerFactory.GetTracer`?
Do we need statistics for those initializers as well?

Reply:
I agree we should use 10,000 spans or so.

initializer stats are useful when trying to optimize the cold startup time - FAAS (Azure functions, Lamba) really care about cold start times.